examples/chat/tui/actions.nim

```diff
@@ -138,6 +138,79 @@ proc action*(self: ChatTUI, event: CreateAccountEvent) {.async, gcsafe,
     self.printResult("Created account:", timestamp)
     self.printResult(fmt"{2.indent()}{name} ({abbrev})", timestamp)
 
+# GetCustomTokensEvent ----------------------------------------------------------
+
+proc action*(self: ChatTUI, event: GetCustomTokensEvent) {.async, gcsafe,
+  nimcall.} =
+  discard
+
+  # if TUI is not ready for output then ignore it
+  if self.outputReady:
+    if event.error != "":
+      self.wprintFormatError(event.timestamp, event.error)
+      return
+
+    let tokens = event.tokens
+    let timestamp = event.timestamp
+    trace "TUI showing tokens from nim-status", tokens=(%tokens)
+
+    if tokens.len > 0:
+      var i = 1
+      self.printResult("Existing tokens:", timestamp)
+      for token in tokens:
+        let
+          name = token.name
+          symbol = token.symbol
+          address = token.address
+
+        self.printResult(fmt"{2.indent()}{name} ({symbol}): {address}", timestamp)
+        i = i + 1
+    else:
+      self.printResult(
+        "No custom tokens added. Add a custom token using `/addcustomtoken <address> <name> <symbol>`.",
```

Comment:
@michaelsbradleyjr has pointed out in one of my earlier PRs that the actions should not be command-dependent, and in that case the actions should not suggest commands.
```suggestion
        "No custom tokens added.",
```

Reply:
Fixed.